deploy/helm/scf/assets/operations/instance_groups/api.yaml

```diff
@@ -92,13 +112,13 @@
   value: https://((deployment-name))-diego-api:8889
 - type: replace
   path: /instance_groups/name=api/jobs/name=cloud_controller_ng/properties/cc/diego?/file_server_url
-  value: http://127.0.0.1:8080
+  value: http://((deployment-name))-api:8080
 - type: replace
   path: /instance_groups/name=api/jobs/name=cloud_controller_ng/properties/cc/diego?/cc_uploader_https_url
-  value: https://127.0.0.1:9091
+  value: https://((deployment-name))-api:9091
```

Comment:
We should remove `127.0.0.1` from the cc_uploader certificate's alternative_names if we are going to do hairpin.